packages/core/package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "@truffle/contract": "^4.2.20",
     "@uma/common": "^1.1.0",
-    "@uma/core-1-1-0": "npm:@uma/core@1.1.0"
+    "@uma/core-1-1-0": "npm:@uma/core@1.1.0",
+    "@uma/core-1-2-0": "npm:@uma/core@1.2.0"
```

Comment:
Is it easy to tell which commit hash differentiates 1.1 from 1.2? and 1.2 from latest?

Reply:
You can look at the repo tags to tell -- that's probably the only way to tell right now.

Reply:
https://github.com/UMAprotocol/protocol/releases